pkg/trace/api/api.go

```diff
@@ -497,6 +508,23 @@ func (r *HTTPReceiver) handleTraces(v Version, w http.ResponseWriter, req *http.
 		}()
 	}
 }
+func extractDroppedP0sHeaders(req *http.Request, ts *info.TagStats) int64 {
+	var droppedP0Traces int64
```

Comment:
Inside this function we are already in the context of "dropped p0s", so there is no need to include that word into the variable name, so why not simply?
```suggestion
	var dropped int64
```